CcWorks/Workers/ReviewWorker.cs

```diff
@@ -82,34 +123,76 @@ await issue.WorkflowTransitionAsync("Submit for Review", new WorkflowTransitionU
 
                 var currentUserIdJson = await GithubHelper.Query("query {viewer {id}}", commonSettings.GithubToken);
                 var userId = currentUserIdJson["viewer"]["id"].Value<string>();
-
                 var nodes = jPullRequest["assignees"]["nodes"] as JArray ?? new JArray();
-                foreach (var node in nodes)
+
+                if (nodes.Any(node => node["id"].Value<string>() == userId))
                 {
-                    if (node["id"].Value<string>() == userId)
-                    {
-                        Console.WriteLine("already assigned");
-                        return;
-                    }
+                    Console.WriteLine("already assigned");
+                }
+                else
+                {
+                    var assignPrMutation = string.Format(MutationAddAssigneePRFormat, jPullRequest["id"].Value<string>(), userId);
+                    await GithubHelper.Query(assignPrMutation, commonSettings.GithubToken);
+                    Console.WriteLine("done");
                 }
 
-                var prId = jPullRequest["id"].Value<string>();
-                var assignPrMutation = @"mutation
-                    {
-                      addAssigneesToAssignable(input: {
-                        assignableId: """ + prId + @""", 
-                        assigneeIds: [""" + userId + @"""]}) { clientMutationId }}";
+                if (settings.AssignReviewLabel)
+                {
+                    await AssignLabelToPR(commonSettings, repoName, settings.ReviewLabelName, jPullRequest);
+                }
+            }
+        }
+
+        private static async Task AssignLabelToPR(CommonSettings commonSettings, string repoName, string reviewLabel, JToken jPullRequest)
+        {
+            Console.Write("Get CRN review label id... ");
 
-                await GithubHelper.Query(assignPrMutation, commonSettings.GithubToken);
+            var getRepoLabels = string.Format(QueryGetRepoLabelsFormat, "trilogy-group", repoName);
+            var repoLabels = await GithubHelper.Query(getRepoLabels, commonSettings.GithubToken);
+            var repoLabelsNode = repoLabels["repository"]["labels"]["nodes"] as JArray;
+            var prId = jPullRequest["id"].Value<string>();
 
+            if (repoLabelsNode == null)
+            {
+                Console.Write("No labels found in repo");
+                return;
+            }
+
+            var crnReviewLabelId = repoLabelsNode.First(
```

Comment:
This code can cause exception, when there are labels in current repository, but there is no label with name `reviewLabel`. Please process this case.